homework2.py

```diff
@@ -0,0 +1,428 @@
+# exercise 5
+'''
+1. write a Python program to find those numbers
+which are divisible by 7 and multiple of 5, between 1500 and 2700 (both included).
+'''
+
+# code up your solution here
+for i in range(1500,2700):
+	if i%7==0 and i%5==0:
+		print(i)
+
+'''
+2. Write a Python program to count the number of even and odd numbers from a series of numbers.
+'''
+num_list = [1,2,3,4,5,6,3,4,1,12,4,6,745,34352,123,6,5,3,3,432,3.1415,5.23]
+odd = []
+even = []
+other = []
+
+for i in num_list:
+    if i%2==1:
+        odd.append(i)
+    elif i%2==0:
+        even.append(i)
+    else:
+        other.append(i)
+
+num_list2 = {'odd':odd, 'even':even, 'other':other}
```

Comment:
you named it `num_list2` but i'd recommend name it `num_dict` or something like this... name implies it's a list but it's actually a dict...